index.js

```diff
@@ -35,6 +36,23 @@ function rehypeReact(options) {
   function h(name, props, children) {
     var component = name
 
+    // Currently, a warning is triggered by react for *any* white space in
+    // tables.
+    // So we remove the pretty lines for now.
+    // See: <https://github.com/facebook/react/pull/7081>.
+    // See: <https://github.com/facebook/react/pull/7515>.
+    // See: <https://github.com/remarkjs/remark-react/issues/64>.
+    if (children && tableElements.indexOf(name) !== -1) {
+        children = children.filter(function (child) {
```

Comment:
I think `npm test` will also format this code, could you do that?

Reply:
@stevemk14ebr Could you run `npm test` on your code to format it matching the project’s style?

Reply:
done!